docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,556 @@
+# Writing Libraries with Cross Platform Tools
+
+By [Phillip Carter](https://github.com/cartermp)
+
+**Some details are subject to change as the toolchain evolves.**
+
+Targeting .NET Core for new libraries can be done a number of ways, depending on the platform(s) you wish to support.  If you cannot or choose not to use Visual Studio to build and test new libraries or wish to use the command line tooling on any Operating System, then this guide is for you.
+
+## Prerequisites
+
+You must have .NET Core installed on your machine.  This guide will focus on command line tooling because they work cross-platform and are what Visual Studio uses internally.  You have two options:
+
+* [DNX](http://aspnet.readthedocs.org/en/latest/getting-started/index.html), which has shipped with Visual Studio 2015 Update 1
+
+* [.NET CLI](http://dotnet.github.io/getting-started/), the tooling which will eventually replace DNX
+
+Either toolchain can be used to build and publish libraries today.
+
+The sections of this document dealing with .NET 4.5 Framework, .NET 4.0 Framework, or Portable Class Libraries (PCLs) require Windows and the .NET Framework installed.  The best way to do this is to [install Visual Studio](https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx).  Alternatively, you can [install .NET Framework 4.6](https://www.microsoft.com/en-us/download/details.aspx?id=48130) as a standalone package.  This will contain all the dependencies required for .NET Framework 4.5, 4.0, and any PCLs you may wish to support.
+
```

Comment:
Should we list the subjects (subsections) you're covering here so the reader can know what to expect to find on this topic and jump to the desired section if they want? Are anchors supported with Markdown?